msautotest/api/expected/ogcapi_api.json

```diff
@@ -1 +1 @@
atures-1.yaml#/components/parameters/datetime"},{"$ref":"http://schemas.opengis.net/ogcapi/features/part2/1.0/openapi/ogcapi-features-2.yaml#/components/parameters/bbox-crs"},{"$ref":"http://schemas.opengis.net/ogcapi/features/part2/1.0/openapi/ogcapi-features-2.yaml#/components/parameters/crs"},{"$ref":"#/components/parameters/offset"},{"description":"The optional limit parameter limits the number of items that are presented in the response document.","explode":false,"in":"query","name":"limit","required":false,"schema":{"default":9,"maximum":15,"minimum":1,"type":"integer"},"style":"form"}],"responses":{"200":{"$ref":"http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/ogcapi-features-1.yaml#/components/responses/Features"},"400":{"$ref":"http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/ogcapi-features-1.yaml#/components/responses/InvalidParameter"},"500":{"$ref":"http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/ogcapi-features-1.yaml#/components/responses/ServerError"}},"summary":"Get Minnesota Population Centers items","tags":["mn_population_centers"]}},"/collections/mn_population_centers/items/{featureId}":{"get":{"description":"This dataset contains cities, townships, reservations, county seats and unorganized Territories in Minnesota, represented with point geometry for cartographic purposes.","operationId":"getmn_population_centersFeature","parameters":[{"$ref":"#/components/parameters/f"},{"$ref":"http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/ogcapi-features-1.yaml#/components/parameters/featureId"}],"responses":{"200":{"$ref":"http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/ogcapi-features-1.yaml#/components/responses/Feature"},"400":{"$ref":"http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/ogcapi-features-1.yaml#/components/responses/InvalidParameter"},"404":{"$ref":"http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/ogcapi-features-1.yaml#/components/responses/NotFound"},"500":{"$ref":"http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/ogcapi-features-1.yaml#/components/responses/ServerError"}},"summary":"Get Minnesota Population Centers item by id","tags":["mn_population_centers"]}},"/conformance":{"get":{"description":"API conformance definition","operationId":"getConformanceDeclaration","parameters":[{"$ref":"#/components/parameters/f"}],"responses":{"200":{"$ref":"http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/ogcapi-features-1.yaml#/components/responses/ConformanceDeclaration"},"400":{"$ref":"http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/ogcapi-features-1.yaml#/components/responses/InvalidParameter"},"500":{"$ref":"http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/ogcapi-features-1.yaml#/components/responses/ServerError"}},"summary":"API conformance definition","tags":["server"]}}},"servers":[{"description":"Server description","url":"http://localhost/cgi-bin/mapserv/OGCAPI_TEST/ogcapi"}],"tags":[]}
```

Comment:
Do crs, crs-bbox, etc... need to be added to the parameters hash in the API JSON output? See https://demo.pygeoapi.io/stable/openapi?f=json for example.

Reply:
They are added in https://github.com/MapServer/MapServer/pull/6893/files#diff-46e72d45fbe7adb5c2df20d7e4a963164b077f5505152841bdd6a2ce022ac42dR1771 as external references, as we do for bbox and datetime

Cf extract of output of ``mapserv -nh  -conf ../etc/mapserv.conf PATH_INFO=/ogcapi.map/ogcapi/api  "QUERY_STRING=f=json" | jq .`` :
```
    "/collections/mn_counties/items": {
      "get": {
        "description": "This is the standard Minnesota State County Boundary dataset.",
        "operationId": "getmn_countiesFeatures",
        "parameters": [
          {
            "$ref": "#/components/parameters/f"
          },
          {
            "$ref": "http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/ogcapi-features-1.yaml#/components/parameters/bbox"
          },
          {
            "$ref": "http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/ogcapi-features-1.yaml#/components/parameters/datetime"
          },
          {
            "$ref": "http://schemas.opengis.net/ogcapi/features/part2/1.0/openapi/ogcapi-features-2.yaml#/components/parameters/bbox-crs"
          },
          {
            "$ref": "http://schemas.opengis.net/ogcapi/features/part2/1.0/openapi/ogcapi-features-2.yaml#/components/parameters/crs"
          },
```